erb_lint.gemspec

```diff
@@ -12,6 +12,10 @@ Gem::Specification.new do |s|
 
   s.files = Dir['lib/**/*.rb']
 
+  s.add_dependency 'html_tokenizer'
+  s.add_dependency 'better_html'
+
   s.add_development_dependency 'rspec'
   s.add_development_dependency 'rubocop'
+  s.add_development_dependency 'activesupport', '~> 5.1'
```

Comment:
This should be `s.add_dependency 'active_support'` and you don't need to specify the version since the features you are using are not 5.1 specific.